### PR TITLE
Change in build class and elasticsearch

### PR DESCRIPTION
**Pull request by adrianfusco**
- Add another status in Job color condition: UNSTABLE
- Change connection to elasticsearch to the only one method that is using it
- Change log comments

### CODE REVIEW

File: cibyl/models/ci/build.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def __str__(self, indent=0, verbosity=0):
             build_str += Colors.blue(f"\n{indent_space}  Status: ")
             if self.status.value == "SUCCESS":
                 build_str += Colors.green(f"{self.status.value}")
-            if self.status.value == "FAILURE":
+            if self.status.value in ("FAILURE", "UNSTABLE"):
```
Comment (cescgina) [inline cibyl/models/ci/build.py:56]: Jenkins associates unstable jobs with yellow. I think we should do the same.

Comment (adrianfusco) [reply cibyl/models/ci/build.py:56]: Oh, I didn't realize that case using elasticsearch. I'll change it, thank u.

**Review by cescgina — APPROVED** [review]
Aside from minor comment, LGTM

File: cibyl/models/ci/build.py (hunk 1 above)
Comment (adrianfusco) [reply cibyl/models/ci/build.py]: Done.

**Review by JoseSilvaHermida — APPROVED** [review]
Looks good to me

**Review by IgnacioPro — APPROVED** [review]
Looks good.

**Comment by cescgina** [comment]
@adrianfusco before merging please check the flake8 output

**Pull request merged**